src/main/java/frc/robot/subsystems/launcher.java

```diff
@@ -61,14 +67,17 @@ boolean pollIrArraySensor(int index) {
     }
   }
 
-public void setDutyout(double percent){
+public void setDutyoutIndex(double percent){
   rightIndexSparkMax.set(percent);
 }
 
-public void launch(double percent) {
+public void SetDutyOutlaunch(double percent) {
   rightLauncherSparkMax.set(percent);
 }
 
+public void SetLauncherVelocity(double setpoint) {
+    pidController.setReference(setpoint, ControlType.kSmartVelocity);
```

Comment:
> Smart Velocity mode - uses same constrains as the Smart Motion mode but outputs velocity instead of position. This is not a motion profiling mode, but rather provides acceleration limiting as opposed to simple ramp rates.

kSmartVelocity is not the same kVelocity.